JM/CF/Debugger/3_Game/CF_Debugger/CF_Debugger.c

```diff
@@ -0,0 +1,199 @@
+/* Example:
+
+void EOnSimulate(...)
+{
+	auto dbg_Helicopters = CF.Debugger.Get("Helicopters", this);
+
+	dbg_Helicopters.Add("Position", GetPosition());
+	dbg_Helicopters.Add("Orientation", GetOrientation());
+
+	CF.Debugger.Display("Helicopters", this, "Velocity", GetVelocity(this));
+}
+
+*/
+
+class CF_Debugger
+{
+	private autoptr array<ref CF_Debugger_Block> m_Blocks;
```

Comment:
I think we should use protected. I see no reason why someone could not inherit their own variant and replace their local CF.Debugger instance at runtime with their own type.